x-pack/legacy/plugins/apm/public/components/shared/KueryBar/index.tsx

```diff
@@ -20,6 +19,7 @@ import { useUrlParams } from '../../../hooks/useUrlParams';
 import { history } from '../../../utils/history';
 import { AutocompleteSuggestion } from '../../../../../../../../src/plugins/data/public';
 import { useDynamicIndexPattern } from '../../../hooks/useDynamicIndexPattern';
+import { esKuery } from '../../../../../../../../src/plugins/data/public';
```

Comment:
I _think_ this is fine but just double-checking: relative imports between plugins are now blessed?

Reply:
Yep, for static code they are okay (both inside and outside of x-pack), as long as they are only importing from the top level `public/index.ts` or `server/index.ts`

The issues with importing from OSS into x-pack have also been resolved; [more explanation on that here](https://github.com/elastic/kibana/pull/45747#discussion_r348118555).

Reply:
also nit: Could combine this import with `AutocompleteSuggestion` in L20